### DISCUSSION TITLE
Slowness of the simulator in building an app in React Native

### DISCUSSION THREAD

**Discussion by paulina-collaguazo** [adobe/react-spectrum#6513] [Show and tell]
Hi, I have a question. I'm creating my first project using React Native, I've followed the tutorial steps, I run npm start, and the server starts up, etc. I choose the iOS device, then it builds and displays the following:
`info Opening app on iOS...
info A dev server is already running for this project on port 8081.
info Found Xcode workspace "Test.xcworkspace"
info No booted devices or simulators found. Launching first available simulator...
info Launching iPhone SE (3rd generation) (iOS 17.2)
info Building (using "xcodebuild -workspace Test.xcworkspace -configuration Debug -scheme")
success Successfully built the app
info Installing `
Then the iOS emulator is activated and only goes up to there, it doesn't go any further, is the process normal?
By the way, I'm running the app on an Apple M3 Pro, and the Xcode version is 17, does that have anything to do with it?

**Comment by reidbarber** [comment]
React Spectrum doesn't support React Native (related issue: https://github.com/adobe/react-spectrum/issues/696). This seems more like a React Native-specific question. If so, maybe try the resources listed here: https://reactnative.dev/community/support
